test/fixtures.rb

```diff
@@ -107,11 +107,11 @@ def call(params)
   end
 end
 
-class MyCustomError < StandardError ; end
```

Comment:
@karimmtarek All the changes in this file are unrelated to the scope of your change, please revert all of them. :smile: